core/iwasm/interpreter/wasm_loader.c

```diff
@@ -11577,6 +11584,16 @@ wasm_loader_prepare_bytecode(WASMModule *module, WASMFunction *func,
                         cell_num = wasm_value_type_cell_num(
                             wasm_type->types[wasm_type->param_count - i - 1]);
                         loader_ctx->frame_offset -= cell_num;
+
+                        if (loader_ctx->frame_offset
+                            < loader_ctx->frame_offset_bottom) {
+                            LOG_DEBUG(
+                                "frame_offset underflow, roll back and "
+                                "let following stack checker report it\n");
```

Comment:
do you have a recipe to trigger this?

Reply:
An unbalanced block will accomplish this. like

```wat
  (func (param f64 v128) (result f64 i64)
    unreachable
    unreachable
    select
    block (type 0) (param f64 v128) (result f64 i64)
    end
    ;; ...
```